### DISCUSSION TITLE
backupNVM over MQTT

### DISCUSSION THREAD

**Discussion by HubertusH** [zwave-js/zwave-js#4332] [Q&A] [answered]
I have triggered an NVM backup via MQTT - "ZJS2M2/_CLIENTS/ZWAVE_GATEWAY-zwavejs2mqtt2/api/backupNVMRaw/set" - which also works and goes through so far. ZWaveJS2MQTT then also report a successful return. The question: WHERE can I find the backup and what is it called?
Thanks.

**Comment by AlCalzone** [comment, marked as answer]
Look in zwavejs2mqtt's store directory. You can access it via zwavejs2mqtt's UI.

**Comment by HubertusH** [comment]
Perfekt - thank you
